tests/Baseline/hilti.hiltic.print.import/output

```diff
@@ -43,16 +43,16 @@ extern void __hlt::Bar::__init_globals(::hilti::rt::Context* ctx) {
 }
 
 extern void __hlt::Bar::__init_module() {
-      __location__("bar.hlt:10:1-10:38");
+      __location__("<...>/bar.hlt:10:1-10:38");
```

Comment:
Yes exactly. I went with this to avoid having to juggle multiple paths in that changed function (which makes it easier to make mistakes) and also since I didn't mind the added verbosity, but I see your point. I changed the patch so baselines stay unchanged.